lib/webchat/config.yaml

```diff
@@ -0,0 +1,48 @@
+  open_url: https://www.tax.service.gov.uk/csp-partials/open/1004
+  availability_url: https://www.tax.service.gov.uk/csp-partials/availability/1004
+- base_path: /government/organisations/hm-revenue-customs/contact/tax-credits-enquiries
+  open_url: https://www.tax.service.gov.uk/csp-partials/open/1016
+  availability_url: https://www.tax.service.gov.uk/csp-partials/availability/1016
+- base_path: /government/organisations/hm-revenue-customs/contact/vat-enquiries
+  open_url: https://www.tax.service.gov.uk/csp-partials/open/1028
+  availability_url: https://www.tax.service.gov.uk/csp-partials/availability/1028
+- base_path: /government/organisations/hm-revenue-customs/contact/customs-international-trade-and-excise-enquiries
+  open_url: https://www.tax.service.gov.uk/csp-partials/open/1034
+  availability_url: https://www.tax.service.gov.uk/csp-partials/availability/1034
+- base_path: /government/organisations/hm-revenue-customs/contact/employer-enquiries
+  open_url: https://www.tax.service.gov.uk/csp-partials/open/1023
+  availability_url: https://www.tax.service.gov.uk/csp-partials/availability/1023
+- base_path: /government/organisations/hm-revenue-customs/contact/online-services-helpdesk
+  open_url: https://www.tax.service.gov.uk/csp-partials/open/1003
+  availability_url: https://www.tax.service.gov.uk/csp-partials/availability/1003
+- base_path: /government/organisations/hm-revenue-customs/contact/charities-and-community-amateur-sports-clubs-cascs
+  open_url: https://www.tax.service.gov.uk/csp-partials/open/1087
+  availability_url: https://www.tax.service.gov.uk/csp-partials/availability/1087
+- base_path: /government/organisations/hm-revenue-customs/contact/enquiries-from-employers-with-expatriate-employees
+  open_url: https://www.tax.service.gov.uk/csp-partials/open/1089
+  availability_url: https://www.tax.service.gov.uk/csp-partials/availability/1089
+- base_path: /government/organisations/hm-revenue-customs/contact/share-schemes-for-employees
+  open_url: https://www.tax.service.gov.uk/csp-partials/open/1088
+  availability_url: https://www.tax.service.gov.uk/csp-partials/availability/1088
+- base_path: /government/organisations/hm-revenue-customs/contact/non-uk-expatriate-employees-expats
+  open_url: https://www.tax.service.gov.uk/csp-partials/open/1089
+  availability_url: https://www.tax.service.gov.uk/csp-partials/availability/1089
+- base_path: /government/organisations/hm-revenue-customs/contact/non-resident-landlords
+  open_url: https://www.tax.service.gov.uk/csp-partials/open/1086
+  availability_url: https://www.tax.service.gov.uk/csp-partials/availability/1086
+- base_path: /government/organisations/hm-passport-office/contact/passport-advice-and-complaints
+  open_url: https://hmpowebchat.klick2contact.com/v03/launcherV3.php?p=HMPO2&d=712&ch=CH&psk=chat_a1&iid=UK&l=en&r=K2C&fcl=0&srbp=0&s=https%3A%2F%2Fhmpowebchat.klick2contact.com%2Fv03
```

Comment:
I think this will need to be changed soon anyway, for now, it's fine. These URLs are generated by the webchat supplier and passed over to us, so something we don't need to care about. I believe the way we have them set up now, it is provider agnostic - therefore, it's the responsibility of the provider to provide a valid link.

Reply:
Just thinking about this, we should probably remove this as Klick2Contact APIs are not returning the required standard at the moment.